modules/templates/ref-creating-templates.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-creating-templates_{context}"]
+= Creating a template as a YAML file
+
+You can create a template by defining a `Template` object as a YAML file.
+
+The `Template` object describes the template and its metadata. It also contains required input variables and a list of actions which are executed by the scaffolding service.
```

Comment:
```suggestion
The `Template` object describes the template and its metadata. It also contains required input variables and a list of actions that are executed by the scaffolding service.
```
Based on IBM style guidance on _which_ vs. _that_ and restrictive vs. nonrestrictive clauses.